.eslintignore

```diff
@@ -0,0 +1,2 @@
+**/test/fixtures/**/node_modules/**
+**/test/work/*
```

Comment:
> There is no technical reason to not support specifying ignore in the package.json, but eslint doesn't support it.

From [the comment right above the one you posted in the linked issue](https://github.com/eslint/eslint/issues/4261#issuecomment-213461874):
> There's also a problem with timing, we need ignore files before we start traversing and we only find config files while we are traversing.

That's what I meant by execution reason.